tasks/components/Label.tsx

```diff
@@ -0,0 +1,17 @@
+import { Span } from '@helpwave/common/components/Span'
+import { tw } from '@helpwave/common/twind'
+import type { PropsWithChildren } from 'react'
+
+export type LabelProps = PropsWithChildren<{
+  name: string,
+  color?: string
+}>;
+
+/**
+ * A component for labelling
+ */
+const Label = ({ name, color = 'hw-primary-400' }: LabelProps) => {
+  return <Span className={tw(`rounded-full text-xs font-bold bg-${color}-background text-${color}-text py-1 px-2`)}>{name}</Span>
```

Comment:
Do something like in the Button component

```
const colors = ['accent', 'accent-secondary', 'positive', 'negative', 'neutral'] as const
type ButtonProps = {color?: typeof colors[number], ...}


<button>
 className={tx('py-2 font-medium focus:outline-none', className, {
      'px-4 text-sm': variant !== 'textButton',

      // primary & {accent, accent-secondary, positive, negative, neutral}
      'text-white bg-hw-primary-400 hover:bg-hw-primary-600 focus:ring-hw-primary-600': color === 'accent',
      'text-white bg-AAA-500    hover:bg-AAA-600    focus:ring-AAA-500': color === 'accent-secondary',
      'text-white bg-hw-positive-400    hover:bg-hw-positive-500    focus:ring-hw-positive-400': color === 'positive',
      'text-white bg-hw-negative-400    hover:bg-hw-negative-500    focus:ring-hw-negative-400': color === 'negative',
      'text-white bg-gray-400    hover:bg-gray-500    focus:ring-gray-400':color === 'neutral', 
    })}
</button>
```

You would also have to extend the tailwind theme in `lib/twind/config.ts` with the new two colors:
hw-label-blue #758ECE
hw-label-pink #CE75A0

The Background color of the label is just the color with 25% opacity

Reply:
Yeah, I have seen this and implemented it this way by adding more color options in the config. But, I didn't find the color code for accent variation. So, I have dropped the code from the commit.

Reply:
@DasProffi Can you help me with the color code for accent variation? I could quickly finish it off. Since I already have code for the above implementation.

Reply:
You do not need to implement any other colors, except:
hw-label-blue #758ECE
hw-label-pink #CE75A0

hw-positive and hw-negative would be nice to have, but aren't required for the label yet

Currently, we do not have a color `accent-scondary`, therefore it's okay to leave it at the placeholder AAA